src/main/java/ch/uzh/ifi/hase/soprafs22/entity/Chat.java

```diff
@@ -1,23 +1,22 @@
 package ch.uzh.ifi.hase.soprafs22.entity;
 
 import javax.persistence.*;
+import java.io.Serializable;
 import java.util.*;
 
 @Entity
 @Table(name = "CHAT")
-public class Chat {
+public class Chat implements Serializable {
 
   @Id
   @GeneratedValue
-  private Long id;
+  @Column(name ="id")
+  private long id;
```

Comment:
Is it possible to give it the same id as the match (since it's a one to one mapping)? Because when the user opens a chat in the client, he doesn't have a chatId (and neither a matchId when I think about it). (Must, Recommendation, Question)

Reply:
yes, found something about it,. will change: https://www.baeldung.com/jpa-one-to-one